packages/database/src/migrations/20210325030144-VaccineDoseTakenByDayVaccineTracking-modifies-data.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+import { insertObject } from '../utilities/migration';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const dashboardGroups = [
+  //  Fiji
+  {
+    code: 'FJ_Covid_Fiji_Country_COVID-19',
+    aggregationEntityType: 'country',
+    dataSourceEntityType: 'district',
+  },
+  {
+    code: 'FJ_Covid_Fiji_District_COVID-19',
+    dataSourceEntityType: 'district',
+  },
+  // // Samoa:
+  {
+    code: 'WS_Covid_Samoa_Country_COVID-19',
+    aggregationEntityType: 'country',
+    dataSourceEntityType: 'village',
+  },
+  // // Nauru:
+  {
+    code: 'NR_Covid_Nauru_Country_COVID-19',
+    aggregationEntityType: 'country',
+    dataSourceEntityType: 'district',
+  },
+  {
+    code: 'NR_Covid_Nauru_District_COVID-19',
+    dataSourceEntityType: 'district',
+  },
+];
+
+const dataElements = [
+  { code: 'COVIDVac4', name: '1st' },
+  { code: 'COVIDVac8', name: '2nd' },
+];
+
+const generateReport = (
+  dashboardGroupCode,
+  aggregationEntityType,
+  dataSourceEntityType,
+  doseName,
+  dataElementCode,
+) => {
+  const entityAggregation = {
+    dataSourceEntityType,
+  };
+  if (aggregationEntityType) entityAggregation.aggregationEntityType = aggregationEntityType;
+
+  return {
+    id: `${dashboardGroupCode}_Num_Of_${doseName}_Vaccine_Dose_Taken`, //
+    dataBuilder: 'sumPerPeriod',
+    dataBuilderConfig: {
+      dataClasses: {
+        value: {
+          codes: [dataElementCode],
+        },
+      },
+      aggregationType: 'FINAL_EACH_DAY',
+      entityAggregation,
+    },
+    viewJson: {
+      name: `Number of people received ${doseName} dose of COVID-19 vaccine by day`,
+      type: 'chart',
+      chartType: 'bar',
+      periodGranularity: 'day',
+    },
+  };
+};
+
+exports.up = async function (db) {
+  for (const { code: dashboardGroupCode, hierarchyLevel } of dashboardGroups) {
```

Comment:
Hmmm would this migration work? It doesn't seem to have `hierarchyLevel` in a dashboard group object?